### DISCUSSION TITLE
Mapbox V11 for react native

### DISCUSSION THREAD

**Discussion by shahbek** [rnmapbox/maps#2993] [Ideas]
V11 of Mapbox introduces new 3d buildings and places, adding this support would see so many people build projects around this. It even comes with sunlight simulation and building shadows casted on the map for an enhance map experience.

**Comment by mfazekas** [comment]
@shahbek RN v11 should be a straightforward migration from v10 api's. So we plan to support v11 in the future.

**Reply by guneyural** [reply]
When do you think you will support v11 and will we be able to use v11 even if it's still in beta?
